src/app/forms/custom-sections/letters/letters.component.spec.ts

```diff
@@ -41,8 +56,13 @@ describe('LettersComponent', () => {
           provide: APP_BASE_HREF,
           useValue: '/',
         },
+        {
+          provide: TechnicalRecordService,
```

Comment:
was this supposed to be `technicalRecordService` ?

Reply:
Yes, I will remove the unnecessary one, it's duplicate.